masonite/routes.py

```diff
@@ -149,10 +149,19 @@ def route(self, route, output):
 
         Returns:
             self
+
+        Raises:
+            RouteException -- Thrown when the incoming route is an empty string.
         """
+        # Checked to make sure the route is not empty
+        try:
+            if route[0] != '/':
+                route = '/' + route
+        except IndexError:
+            raise RouteException("Undefined Route")
```

Comment:
whats all this doing? All that to make sure the route is not empty? can't you just do `if not route` to check for empty?

Reply:
True.

Reply:
All I added was the try/catch. The rest of it is what was there.

Reply:
oh i see now

Reply:
I am not too happy with the fix, so this is all prone to change.

Reply:
I think catching that edge case is important. I can see causing issues for the `"/"` routes. We all have fat fingers and can easily code `""` instead.

Reply:
yes plus the bottom one is valid in 2.2. Hence why you see `route = '/' + route` to begin with since its putting one there for you when you forget